public/js/util.js

```diff
@@ -0,0 +1,229 @@
+var firstFloor = document.getElementById("floor1");
+var secondFloor = document.getElementById("floor2");
+var modal = document.querySelector('.modal');
+var modalOverlay = document.querySelector('.modal-overlay');
+
+var events;
+
+// An AJAX request to get the data from the event-data.json file and parsing it
+$.getJSON({
+    url: "./event-data.json",
+    async: false
+}, function(data) {
+    events = data;
+});
+
+// Transforming the start and end times into date objects
+for (event of events) {
+    var eventStartTime = event.startTime;
+    var eventEndTime = event.endTime;
+    event.startTime = new Date("March 2, 2019" + eventStartTime);
```

Comment:
This shouldn't be hardcoded – at the very least, make a constant in the JS file, but the better option is to encode it into the JSON file since this will break for multi-day events (like horizons)